device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/hwsku.json

```diff
@@ -1,100 +1,132 @@
 {
     "interfaces": {
         "Ethernet0": {
-            "default_brkout_mode": "1x400G[200G,100G,50G,40G,25G,10G,1G]"
```

Comment:
let's only add for MSFT specific SKU which will enable IM feature, for ACS-MSN4700 for now we will not enable IM, so no need to add.

Reply:
Done